### ISSUE TITLE
Lint rules to mark missing type annotations

### ISSUE THREAD

**Issue by odub** [zertosh/eslint-plugin-flow-vars#1]
Hey,

For fixing the false positive problem with eslint shouting about `no-undef` and `no-unused-vars` this plugin is working great. Much appreciated!

I wonder what it would take to get eslint to check for untyped arguments and return values? Does that fall within the scope of this plugin?

For example, code like this…

``` javascript
function multiplyByTwo(number) {
  return number * 2;
}
```

… would run against these additional rules …

``` json
{
 "flow-vars/no-argument-type": 2,
 "flow-vars/no-return-type": 2
}
```

… with this result:

```
1:15  error  Function arguments must have type annotations  flow-vars/no-argument-type
1:19  error  Function return value must have a type annotation  flow-vars/no-return-type
```

**Comment by zertosh** [comment]
This plugin is more about suppressing the warnings, but I'd welcome a PR with that functionality

**Comment by LegNeato** [comment]
I think https://github.com/gajus/eslint-plugin-flowtype already handles that FYI.

**Comment by zertosh** [comment]
I've [deprecated](https://github.com/zertosh/eslint-plugin-flow-vars#this-project-is-deprecated) the project in favor of [eslint-plugin-flowtype](https://github.com/gajus/eslint-plugin-flowtype). I ported the exact same functionality over since it made for sense for there to be only one plugin for flow related linting.

I recommend that you re-file the issue there.
